pkg/scheduler/cache/event_handlers.go

```diff
@@ -129,8 +129,25 @@ func (sc *SchedulerCache) updateTask(oldTask, newTask *schedulingapi.TaskInfo) e
 	return sc.addTask(newTask)
 }
 
+func (sc *SchedulerCache) allocatedPodInCache(pod *v1.Pod) bool {
+	pi := schedulingapi.NewTaskInfo(pod)
+
+	if job, found := sc.Jobs[pi.Job]; found {
+		if t,found := job.Tasks[pi.UID]; found {
+			return schedulingapi.AllocatedStatus(t.Status)
+		}
+	}
+
+	return false
+}
+
```

Comment:
I'm OK about the funciton overall. Sugget add logs and comments for it.

Reply:
Add some log when ignore the update event